joern-cli/frontends/javasrc2cpg/src/main/scala/io/joern/javasrc2cpg/passes/AstCreator.scala

```diff
@@ -429,20 +428,17 @@ class AstCreator(filename: String, javaParserAst: CompilationUnit, global: Globa
   private def identifierForTypeParameter(typeParameter: TypeParameter): NewIdentifier = {
     val name = typeParameter.getNameAsString
     val typeFullName = typeParameter.getTypeBound.asScala.headOption
-      .flatMap { bound =>
-        typeInfoCalc.fullName(bound)
-      }
+      .flatMap(typeInfoCalc.fullName)
       .getOrElse(TypeConstants.Object)
 
     identifierNode(name, typeFullName)
   }
 
   private def identifierForResolvedTypeParameter(typeParameter: ResolvedTypeParameterDeclaration): NewIdentifier = {
     val name = typeParameter.getName
-    val typeFullName = Try(typeParameter.getUpperBound) match {
-      case Success(upperBound) => typeInfoCalc.fullName(upperBound)
-      case Failure(_)          => TypeConstants.Object
-    }
+    val typeFullName = Try(typeParameter.getUpperBound)
+      .map(typeInfoCalc.fullName)
```

Comment:
Same as above. You are catching exception from the `map` now and silently ignore them. Is that intended?